src/utils/Renderers.tsx

```diff
@@ -202,10 +202,44 @@ export const createWaypointsTableBody = (
   );
 };
 
+const getFilterPredicate = (state: Types.state) => {
+  const isSearchBarToggled = state.useSearchBar;
+  const isVisibilityToggled = state.useMarkerVisibility;
+  return isSearchBarToggled
```

Comment:
No nested ternaries please.

Reply:
is it because nested ternaries can get unreadable quickly? 

Would the following if, else if, else statements be better compared to the nested ternary?
```
  if (searchBarToggled) {
    return doesCombinedRowIncludeQuery(state);
  } else if (visibilityToggled) {
    return isMarkerVisible;
  } else {
    return noFilter;
  }
```

Reply:
Yes, that looks much better, and yes, I find nested ternaries to be very hard to read.